cross/toolchain.cmake

```diff
@@ -119,35 +84,6 @@ elseif(TARGET_ARCH_NAME STREQUAL "x86")
   add_compile_options(-Wno-error=unused-command-line-argument)
 endif()
 
-# Set LLDB include and library paths
```

Comment:
This needs to stay - the diagnostics repo uses it (see my comment above on updating and using the eng/common/cross/toolchain.cmake)

Reply:
Any particular reason we couldn't scope this to a version of this file that exists only in the diagnostics repo? We'd have the shared file with the options used by everyone, and the diagnostics repo would have their toolchain file include the common one and also have this.

Reply:
The point is -we should eventually migrate all repos to use eng/common/cross/toolchain.cmake. So we will be bringing changes from this file over to that one and if we delete it now, it may accidentally get deleted from the other one. In other words, let's consider this cross/toolchain.cmake to be what we'll copy into the eng/common/cross/toolchain.cmake later.

Reply:
Sounds good to me.